administration.html

```diff
@@ -0,0 +1,272 @@
+                  fill-rule="nonzero"/>
+        </symbol>
+        <symbol viewBox="0 0 32 32" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round"
+                stroke-miterlimit="1.414" id="github" xmlns="http://www.w3.org/2000/svg">
+            <path d="M16 1.371c-8.284 0-15 6.715-15 15 0 6.627 4.298 12.25 10.258 14.233.75.138 1.026-.326 1.026-.722 0-.357-.014-1.54-.021-2.793-4.174.907-5.054-1.77-5.054-1.77-.682-1.733-1.665-2.195-1.665-2.195-1.361-.931.103-.912.103-.912 1.506.106 2.299 1.546 2.299 1.546 1.338 2.293 3.509 1.63 4.365 1.247.134-.969.523-1.631.952-2.006-3.331-.379-6.834-1.666-6.834-7.413 0-1.638.586-2.976 1.546-4.027-.156-.378-.669-1.903.145-3.969 0 0 1.26-.403 4.126 1.537a14.453 14.453 0 0 1 3.755-.505c1.274.006 2.558.173 3.757.505 2.864-1.94 4.121-1.537 4.121-1.537.816 2.066.303 3.591.147 3.969.962 1.05 1.544 2.389 1.544 4.027 0 5.761-3.509 7.029-6.849 7.401.538.466 1.017 1.379 1.017 2.778 0 2.007-.018 3.623-.018 4.117 0 .399.27.867 1.03.72C26.707 28.616 31 22.996 31 16.371c0-8.285-6.716-15-15-15z"
+                  fill-rule="nonzero"/>
+        </symbol>
+        <symbol viewBox="0 0 32 32" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round"
+                stroke-miterlimit="1.414" id="envelope" xmlns="http://www.w3.org/2000/svg">
+            <g fill-rule="nonzero">
+                <path d="M30.1 5.419c-9.355-.002-18.733-.005-28.1-.005A1.06 1.06 0 0 0 .975 6.439v19.122A1.06 1.06 0 0 0 2 26.586h28a1.061 1.061 0 0 0 1.025-1.025V6.439a1.056 1.056 0 0 0-.925-1.02zM3.025 7.464h25.95v17.072H3.025V7.464z"/>
+                <path d="M30.06 9.513c.933.098 1.382 1.395.393 1.945L16.54 18.287c-.438.188-.479.178-.893 0L1.733 11.458c-1.743-.968-.065-2.254.894-1.842l13.466 6.61 13.562-6.651c.3-.094.312-.062.405-.062z"/>
+            </g>
+        </symbol>
+    </defs>
+</svg>
+<header class="navbar" role="navigation">
+    <div class="navbar__container">
+        <a class="navbar__home" href="index.html">
+            <img class="navbar__logo" src="img/logo-marianne.svg"
+                 alt="particulier.beta.gouv.fr"/><span class="navbar__domain">particulier</span><img
+                src="img/pointapigouvfr.svg" class="navbar__gouvfr" alt=".api.gouv.fr"/>
+        </a>
+
+        <nav>
+            <ul class="nav__links">
+                <li class="nav__item"><a href="https://api.gouv.fr/apropos">À propos</a></li>
+                <li class="nav__item"><a
+                        href="mailto:contact@particulier.api.gouv.fr?subject=Contact%20via%20particulier.api.gouv.fr">Nous
+                    contacter</a></li>
+            </ul>
+        </nav>
+    </div>
+</header>
+
+<div class="hero" role="banner">
+    <div class="hero__container administration">
+        <div class="row">
+            <div class="column ">
+                <h1>Pas de file d'attente dans ma mairie</h1>
```

Comment:
Should it be " Pas de file d'attente dans ma mairie !" ?
or
"En finir avec les files d'attente dans ma mairie !"